radarlabs/react-native-radar
# Adding Typescript support

Adding Typescript to repo

Adding types to mobile react native code, web react native typing not in scope of this PR.
Current implementation design: adding typing to the mobile module and seperating the web module into another named export. This will introduce a breaking change for existing web module users. Minor version bump and short migration guide needed. 

QA:
There is a playground mode in this branch of waypoint: https://github.com/radarlabs/waypoint/tree/typingQA
Enable super user mode and click the enter playground button to enter the mode
There are buttons which perform calls to the SDK which prints out the output onto the screen.
The typing should be available via vscode
